code/datums/ruins/space.dm

```diff
@@ -0,0 +1,83 @@
+/datum/map_template/ruin/space
+	prefix = "_maps/RandomRuins/SpaceRuins/"
+	cost = 1
+
+/datum/map_template/ruin/space/zoo
+	id = "zoo"
+	suffix = "abandonedzoo.dmm"
+
+/datum/map_template/ruin/space/asteroid1
+	id = "asteroid1"
+	suffix = "asteroid1.dmm"
+
+/datum/map_template/ruin/space/asteroid2
+	id = "asteroid2"
+	suffix = "asteroid2.dmm"
+
+/datum/map_template/ruin/space/asteroid3
+	id = "asteroid3"
+	suffix = "asteroid3.dmm"
+
+/datum/map_template/ruin/space/
```

Comment:
I don't know anything about space ruins, so I just left the name and descriptions blank.